test/cppsuite/tests/run.cpp

```diff
@@ -43,6 +43,9 @@
 #include "search_near_02.cpp"
 #include "search_near_03.cpp"
 #include "test_template.cpp"
+#include "cursor_bound_04.cpp"
```

Comment:
Wish we could automate that too, would be great to sort alphabetically all the changes in this file.

Reply:
3396073cb